http/service.go

```diff
@@ -1513,6 +1518,47 @@ func (s *Service) handlePprof(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Pattern should start with "all:" prefix in order to
+// include files and directories beginning with "." or "_".
+//go:embed all:ui/out/*
+var staticAssets embed.FS
```

Comment:
ui/out directory is embedded as virtual file system at build time.

Reply:
That's cool, definitely looking forward to learning more about that.